File: app/src/main/java/ru/vegax/xavier/a3test/photos/PhotosActivityViewModel.java
package ru.vegax.xavier.a3test.photos;

import android.arch.lifecycle.LiveData;
import android.arch.lifecycle.MutableLiveData;
import android.arch.lifecycle.ViewModel;
import android.support.annotation.NonNull;

import java.util.List;

import ru.vegax.xavier.a3test.models.User;
import ru.vegax.xavier.a3test.models.UserPhoto;
import ru.vegax.xavier.a3test.models.UserPhotoWrapper;
import ru.vegax.xavier.a3test.repositories.UsersRepository;

class PhotosActivityViewModel extends ViewModel {

    private MutableLiveData<UserPhotoWrapper> mUserPhotos;

    void init(){
        if(mUserPhotos != null){
            return;
        }
        UsersRepository repo = UsersRepository.getInstance();
        mUserPhotos = repo.getUserPhoto();
    }

    @NonNull LiveData<UserPhotoWrapper> getUserPhotos() {
        return mUserPhotos;
    }

}

File: app/src/main/java/ru/vegax/xavier/a3test/photos/PhotosActivityViewModel.kt
package ru.vegax.xavier.a3test.photos

import android.arch.lifecycle.LiveData
import android.arch.lifecycle.MutableLiveData
import android.arch.lifecycle.ViewModel

import ru.vegax.xavier.a3test.models.User
import ru.vegax.xavier.a3test.models.UserPhoto
import ru.vegax.xavier.a3test.models.UserPhotoWrapper
import ru.vegax.xavier.a3test.repositories.UsersRepository

internal class PhotosActivityViewModel : ViewModel() {

    private var mUserPhotos: MutableLiveData<UserPhotoWrapper>? = null

    val userPhotos: LiveData<UserPhotoWrapper>?
        get() = mUserPhotos

    fun init() {
        if (mUserPhotos != null) {
            return
        }
        val repo = UsersRepository.getInstance()
        mUserPhotos = repo.userPhoto
    }

}
